Data/User.json

```diff
@@ -0,0 +1 @@
+{"roles":["Member","Admin"],"name":"John Deo","admin":true,"email":"john.doe@example.com"}
```

Comment:
This brings up the warning "No newline at end of file". I believe this can be amended simply by adding another line underneath. Apparently, this could bring up unexpected behavior. I'll amend this my side as it should be a quick fix.